fcrepo-webapp/src/main/webapp/static/constraints/RequestWithAclLinkHeaderException.rdf

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-stylesheet type="text/xsl" href="owl2html.xsl"?>
+
+<!DOCTYPE rdf:RDF [
+    <!ENTITY owl "http://www.w3.org/2002/07/owl#" >
+    <!ENTITY xsd "http://www.w3.org/2001/XMLSchema#" >
+    <!ENTITY rdfs "http://www.w3.org/2000/01/rdf-schema#" >
+    <!ENTITY rdf "http://www.w3.org/1999/02/22-rdf-syntax-ns#" >
+]>
+
+
+<rdf:RDF
+     xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#"
+     xmlns:owl="http://www.w3.org/2002/07/owl#"
+     xmlns:xsd="http://www.w3.org/2001/XMLSchema#"
+     xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#">
+    <owl:Ontology rdf:about="static/constraints/CannotCreateResourceWithAclException">
+        <rdfs:label xml:lang="en">Request With Acl Link Header Exception</rdfs:label>
+        <rdfs:comment xml:lang="en">
+          Fedora will not handle request with the specified LDP-RS as the ACL in the request.
```

Comment:
change message to: 

Fedora cannot handle requests containing Link headers where rel="acl".  ACL locations are system defined and cannot be altered.